### ISSUE TITLE
Добавить логи

### ISSUE THREAD

**Issue by bobrolov** [MihailBelyaev/RustTestProject#2]
Чтобы программа сообщала в лог, если у нее получилось\не получилось прочитать из монги
А так же, чтобы программа писала в логи успешные и неуспешные запросы реста

**Comment by BogdanovAleksei** [comment]
Удобно для такой задачи использовать [tracing](https://docs.rs/tracing/latest/tracing/)
И если получится, то при запуске приложения внутри контейнера нужно будет отключить цветное логирование

**Comment by BogdanovAleksei** [comment]
Возможно хорошим дополнением было бы задавать уровень логирования, как аргумент запуска контейнера

**Comment by MihailBelyaev** [comment]
Просто на местах выжных операций добавить event!  с соответствующим уровнем?

**Comment by bobrolov** [comment]
Да, можно так, но удобнее, через макросы с уже заданным уровнем
tracing::info!/warn!/error!("Text of the message");
